src/client/datascience/interactive-common/intellisense/conversion.ts

```diff
@@ -262,14 +262,16 @@ function convertToMonacoMarkdown(
         | vscode.MarkedString
         | vscode.MarkedString[]
 ): monacoEditor.IMarkdownString[] {
-    if (strings.hasOwnProperty('kind')) {
+    // tslint:disable-next-line: no-any
+    if (strings.hasOwnProperty('kind') || (strings as any).kind) {
```

Comment:
The new condition is sufficient `(strings as any).kind`, won't need `hasOwnProperty`

Reply:
That's not true as (strings as any).kind will resolve as false if the value is 0 or false. I want it to continue to work if the property happens to be there.

Reply:
Doesn't 'in' keyword work here? 'kind' in strings.

Reply:
Let me double check

Reply:
Yes that seems to work. Not sure what the difference is. I assumed 'in' used hasOwnProperty under the covers.

Reply:
@rchiodo Heh, I only know because I'd been going through a JS basics book (felt like there was stuff that I didn't know jumping directly into TS) and just read the chapter covering JS object. hasOwnProperty is only checking the top level object prototype for the property, so if it's inherited it won't show up. 'in' keywork check the entire property chain. So for an object hasOwnProperty('toString') return false, since that method is in the base object prototype.